services.yaml

```diff
@@ -1573,17 +1579,17 @@ paths:
           $ref: "#/responses/UnauthorizedError"
         403:
           $ref: "#/responses/ForbiddenError"
-  /services/{service_id}/views/{view_id}/events:
+  /services/{env_id}/views/{view_id}/events:
     get:
       tags:
         - Views
         - Events
       summary: Fetch events details
-      description: Fetch events data in a selected timeframe
+      description: Fetch events data for a target timeframe, application / server / deployment set.
```

Comment:
not sure i got the addition of application / server / deployment. do you mean the ability to filter (which as i see not exists)?

Reply:
it does exist:
```
  - in: query
           name: server	           name: server
           type: string	           type: string
           description: Server(s) added to the filter	           description: Server(s) added to the filter
         - in: query	         - in: query
           name: app	           name: app
           type: string	           type: string
           description: Application(s) added to the filter	           description: Application(s) added to the filter
         - in: query	         - in: query
           name: deployment	           name: deployment
           type: string	           type: string
           description: Deployment(s) added to the filter	           description: Deployment(s) added to the filter
         - in: query	         - in: query
```